src/aiida_quantumespresso/workflows/ph/base.py

```diff
@@ -22,8 +22,9 @@ class PhBaseWorkChain(ProtocolMixin, BaseRestartWorkChain):
 
     defaults = AttributeDict({
         'delta_factor_max_seconds': 0.95,
-        'delta_factor_alpha_mix': 0.90,
-        'alpha_mix': 0.70,
+        'delta_factor_alpha_mix': 0.65,
```

Comment:
Just out of curiosity: how did you decide on this value?

I recently tested reducing the `delta_factor_mixing_beta` for the `PwBaseWorkChain` on a set of failed SCF runs and didn't really help much (surprisingly, to be honest). There we do start from `mixing_beta` 0.4 (also see my other comment below).

Reply:
Yes, probably better to start from `alpha_mix(1) = 0.4` and work they way down. The delta factor I chose it in a way to go pretty fast below 0.1, so one can explore different mixings in the standard 5 max iterations.
Do you have any better idea?

Reply:
Nothing based on hard facts. In https://github.com/aiidateam/aiida-quantumespresso/pull/987 my thinking was now to stick to three options by default: 0.4 -> 0.2 -> 0.1. So basically make the delta factor 0.5. If one of those three doesn't work, all hope is probably lost for this strategy?